nzedb/Contents.php

```diff
@@ -214,37 +282,49 @@ public function data_getAll()
 	 */
 	public function data_getAllButFront()
 	{
-		return $this->pdo->query(sprintf("SELECT * FROM page_contents WHERE id != 1 ORDER BY contenttype, COALESCE(ordinal, 1000000)"));
+		return $this->pdo->query(sprintf('SELECT * FROM page_contents WHERE id != 1 ORDER BY contenttype, COALESCE(ordinal, 1000000)'));
 	}
 
+	/**
+	 * @param $id
+	 * @param $role
+	 *
+	 * @return array|bool
```

Comment:
->queryOneRow() can only return an array or false. Here and there should both be updated.